src/createThemeFile.ts

```diff
@@ -0,0 +1,48 @@
+import renderFiles from '@dojo/cli-create-app/renderFiles';
```

Comment:
I'm not sure that we should be creating dependencies between different commands like this. Looks like you copied this approach from `cli-create-widget`. As this is now used in multiple places is it possible that `renderFiles` and the `template` functionality it uses should be moved elsewhere? A `cli-util` package or something? Worth raising an issue for.

Reply:
True I did find it unusual to consume a repo in that manner, but yep I copied the approach so though it was an established pattern! Just two q's, did you mean raise an issue in this repository? I did that: https://github.com/dojo/cli-create-theme/issues/2

Also: is that something we should do now, before this PR is merged?

Reply:
Matt replied super quickly! https://github.com/dojo/cli-create-theme/issues/2#issuecomment-349966135 and is saying we have a helper in the cli repo which can be used for that, so still wondering if you think we should do that before merging this pr

Reply:
PR submitted! https://github.com/dojo/cli/pull/180